advisories/a79281.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Technical Advisory 79281
+summary: Importing duplicate keys can cause violations of UNIQUE constrants
+toc: true
+docs_area: releases
+---
+
+Publication date: April 12, 2022
+
+## Description
+
+In a multi-region database, calling [`IMPORT INTO`](../v22.1/import-into.html) on a [`REGIONAL BY ROW` table](../v22.1/regional-tables.html) could violate [`UNIQUE`](../v22.1/unique.html) constraints, including [`PRIMARY KEY`](../v22.1/primary-key.html) constraints, resulting in duplicate keys.
+
+Affected versions: v21.2.0-v21.2.7
+
+## Statement
+
+This is resolved in CockroachDB by [PR 79293](https://github.com/cockroachdb/cockroach/pull/79293).
+The fix has been applied to the maintenance release of [CockroachDB v21.2.8](../releases/v21.2.html#v21-2-8).
+
+This public issue is tracked by [79281](https://github.com/cockroachdb/cockroach/issues/79281).
+
+## Mitigation
+
+Users of CockroachDB v21.2.0 to v21.2.7 who have used [`IMPORT INTO`](../v22.1/import-into.html) should audit their [`REGIONAL BY ROW`](../v22.1/regional-tables.html) tables with `UNIQUE` constraints to ensure these tables do not contain `UNIQUE` constraint violations.
+
+Users should also upgrade to v21.2.8 or a later version.
+
+### v21.2.6 or later
+
+Users of CockroachDB v21.2.6 and later have access to the following [built-in functions](../v22.1/functions-and-operators.html), which can be used to easily validate unique constraints:
```

Comment:
Maybe cap and inline code on "unique" here to match elsewhere?

Reply:
And there is a later one too, so perhaps a quick double check on these in the file?